allennlp/service/server_sanic.py

```diff
@@ -8,34 +8,36 @@ def run(port: int) -> None:
     print("Starting a sanic server on port {}.".format(port))
     app = make_app()
     # TODO(joelgrus): make this configurable
-    app.servables = ServableCollection.default()
+    app.predictors = PredictorCollection.default()
```

Comment:
right now there is a `PredictorCollection.from_params()` that's not implemented yet. once I implement that, I plan to delete `default()` and do what you suggest. I'll do that in another PR.